pages/common/encfs.md

```diff
@@ -0,0 +1,24 @@
+# encfs
+
+> Encrypted virtual filesystem.
+> More information: <https://github.com/vgough/encfs>.
+
+- Initialize or mount an encrypted filesystem:
+
+`encfs {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Initialize an encrypted filesystem with standard settings:
+
+`encfs --standard {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
```

Comment:
```suggestion
`encfs --standard {{path/to/cipher_dir}} {{path/to/mount_point}}`
```

Reply:
The path _must_ be absolute here, should I specify it in an other way ?

Reply:
Then definitely leave it as is.  We prefer relative if possible, otherwise absolute.

Reply:
@michaeldel should the path in the last examplie also be absolute?

Reply:
@marchersimon `fusermount` can work with relative paths.